net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils gnome2-utils perl-module vcs-snapshot
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="https://trizen.googlecode.com"
+SRC_URI="https://github.com/trizen/youtube-viewer/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+RDEPEND="
+	>=dev-lang/perl-5.16.0
```

Comment:
The eclass already pulls perl as a dependency: https://github.com/gentoo/gentoo/blob/master/eclass/perl-module.eclass#L81 Do we really need this here?